src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1074,14 +1074,14 @@ private function addPhpErrorsSection(ArrayNodeDefinition $rootNode)
                     ->info('PHP errors handling configuration')
                     ->addDefaultsIfNotSet()
                     ->children()
-                        ->scalarNode('log')
+                        ->variableNode('log')
                             ->info('Use the application logger instead of the PHP logger for logging PHP errors.')
-                            ->example('"true" to use the default configuration: log all errors. "false" to disable. An integer bit field of E_* constants.')
+                            ->example('"true" to use the default configuration: log all errors. "false" to disable. An integer bit field of E_* constants. A mapping from integer (E_* constant) to a log level')
```

Comment:
`An integer bit field of E_* constants, or an array mapping E_* constants to a log levels.`